scout/static/scout/js/test/filterTest.js

```diff
@@ -0,0 +1,412 @@
+var filter = require('../filter');
```

Comment:
You should probably do something like `Filter = require('../filter').Filter` since you don't appear to use anything other than Filter from filter.js.